deepchecks/nlp/utils/text_embeddings.py

```diff
@@ -30,6 +68,8 @@ def calculate_builtin_embeddings(text: np.array, model: str = 'miniLM',
         by using the command openai.api_key = YOUR_API_KEY
```

Comment:
Need to add info on that process we are doing + ref link to open ai cook book

Reply:
Added